internal/app/machined/pkg/controllers/block/mount.go

```diff
@@ -495,6 +495,7 @@ func (ctrl *MountController) handleDiskMountOperation(
 
 		if mountRequest.TypedSpec().ReadOnly {
 			opts = append(opts, mount.WithReadOnly())
```

Comment:
this not needed then?

Reply:
my understanding is that opening fs as ro mounting as read-only are different concepts

Mateusz, what I meant is that can we in mount package if I say `mount.ReadOnly` enforce fsopen with `ro`?

Reply:
> my understanding is that opening fs as ro mounting as read-only are different concepts
> 
> Mateusz, what I meant is that can we in mount package if I say `mount.ReadOnly` enforce fsopen with `ro`?

ahh I get it now, we need both I guess